src/test/mir-opt/const_allocation3/rustc.main.ConstProp.after.mir

```diff
@@ -27,21 +27,25 @@ fn  main() -> () {
 }
 
 alloc2 (static: FOO, size: 8, align: 8) {
-     ├──────alloc8+0───────┤
+     ├──────alloc9+0───────┤
 }
 
-alloc8 (size: 140, align: 1) {
+alloc9 (size: 150, align: 1) {
     0x00 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab
-    0x10 │ ab ab ab ab ab ab ab ab ab ab ab ab ├──alloc6+0──
+    0x10 │ ab ab ab ab ab ab ab ab ab ab ab ab ├─alloc6+0─
     0x20 │ ──────────┤ 01 ef cd ab 00 00 00 00 00 00 00 00
     0x30 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
     0x40 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
     0x50 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
     0x60 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
     0x70 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
-    0x80 │ 00 00 00 00 00 00 00 00 00 00 00 00
+    0x80 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 ├alloc8+0 (8 bytes)
+    0x90 │ ────────────────┤
 }
 
+alloc8 (fn: main)
+
+
```

Comment:
Looks like we usually have one empty line between allocations, but here it's two?